chalk/__init__.py

```diff
@@ -5,10 +5,10 @@
 
 from chalk.core import Diagram, Empty, Primitive
 from chalk.shape import Circle, Rectangle, Path, Text, Image
-from chalk.point import Point
+from chalk.point import Point, Vector
 from chalk.trail import Trail
 
-ignore = [Trail]
+ignore = [Trail, Vector]
```

Comment:
I was curious: how is this `ignore` variable used?

Reply:
Oh I just added that to make `flake` not throw an error, but have Vector available to the user.

Reply:
Ah, I see, thanks for the explanation!